rfcs/text/0000-multi-data-collection-dataset-processing.md

```diff
@@ -0,0 +1,115 @@
+
+### Identifying data to co-process
+Pipelines that require grouping data from multiple sequencing assays (co-processing) subscribe to *PROJECT_SUBMISSIONS* events instead of per-bundle events. Unlike with the assay bundle per pipeline run, this makes the pipeline framework responsible for collecting assay bundles into pipeline runs.  Pipelines that do not require co-processing continue to subscribe to assay bundle events and are unaffected.
+
+For full background and a detailed proposal on how these libraries are represented in the metadata see the RFC (Representing sequencing library preparations in the HCA DCP metadata standard)[https://github.com/HumanCellAtlas/dcp-community/blob/mfreeberg-rfc-bundle-definitions/rfcs/text/0000-bundle-definition.md].
+
+The *data group* submission event does not define bundles to be co-processed, it indicates that data to be co-processed is contained within the *data group* and that that group fulfills version-completeness requirements. Co-processing will be initiated by pipelines based on a well-defined set of rules that is data modality-specific.  The partition of a submission *data group* into co-processing units may require multiple parameters. For instance, with 10X V2 3’ scRNA-seq is grouped by library, however, a submission my contain non-10X data modalities that need to be processed separately.
+
+An analysis pipeline event handler component will be created that is responsible for partitioning a *data group* into co-processing units, dispatching the pipelines, and tracking completion. 
+
+
+![Figure 2](/rfcs/images/0000-multi-data-collection-data-processing-images/flow-of-information.png)
+
+#### Updates to *data groups*
+Updates to PROJECT_SUBMISSIONS must result in updating of the associated *data groups* to trigger reprocessing. When part or the entirety of a project submission is updated the associated data group must be identified and updated with the new bundle versions if bundles have been replaced and/or with additional new bundles if bundles have been added. This is the responsibility of Ingest. Analysis must handle the update to the *data group* by initiating only the pipelines that have been affected by the update and ensure that the output bundles of this analysis are updates to existing bundles.
+
+### Unresolved Questions
```

Comment:
I think it is very important to think about how this will stretch beyond PROJECT_SUBMISSION. It doesn't need to be high detail but I would need a warm feeling that we are not heading down a development path that can't cater to later requirements.

Reply:
As below, on reconsideration I don't think is too important to answer now as I suspect PROJECT_SUBMISSION is needed in pretty much all circumstances.